src/SprykerEco/Zed/Econda/Business/EcondaFacadeInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda\Business;
+
+use Generated\Shared\Transfer\BatchResultTransfer;
+use Generated\Shared\Transfer\LocaleTransfer;
+use SprykerEco\Zed\Econda\Business\Exporter\Writer\File\FileWriterInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface EcondaFacadeInterface
```

Comment:
why does the outputinterface come through the facade? also the `FileWriterInterface` looks very-very strange

Reply:
Not sure. I'm not that person who implemented this. You can tell how it should be and I will update the code. Thanks